gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/operators/diff/functions/DiffPerElement.java

```diff
@@ -47,8 +47,16 @@
    */
   private final TemporalPredicate second;
 
+  /**
+   * Specifies the time dimension that will be considered by the operator.
+   * Will be {@link TimeDimension#VALID_TIME} by default.
+   */
+  private TimeDimension dimension = TimeDimension.VALID_TIME;
```

Comment:
The `ReadFields` FunctionAnnotation will be wrong, if this is not the default. Consider removing it completely, since flink is pretty good without annotations.